### ISSUE TITLE
Investigate using zkcli to set ACLs

### ISSUE THREAD

**Issue by stevendborrelli** [CiscoCloud/microservices-infrastructure#329]
Looks like this zkcli is starting to get ACL support:

https://github.com/outbrain/zookeepercli/pull/8

Our way of setting ZK ACL's is hackish, this might be a better solution. I looked at this client before but it didn't support ACLs (and the java zkcli did not work well).

**Comment by keithchambers** [comment]
Is this why every third `vagrant up` fails for me with:
```
NOTIFIED: [zookeeper | register zookeeper service] **************************** 
failed: [default] => {"failed": true}
msg: Job for zookeeper.service failed. See 'systemctl status zookeeper.service' and 'journalctl -xn' for details.


FATAL: all hosts have already failed -- aborting

TASK: [mesos | add mesosphere repository] ************************************* 
FATAL: no hosts matched or all hosts have already failed -- aborting
```

**Comment by ChrisAubuchon** [comment]
Can you look at /etc/systemd/system/multi-user.target.wants/zookeeper.service and see if the data volume is using the `tianon/true` container? There is an issue with that container and Docker 1.6 that caused us to create the `ciscocloud/data-volume:0.1` container to use instead.

**Comment by stevendborrelli** [comment]
That pull request was merged, so we'll be able to test it for 0.4.

**Comment by stevendborrelli** [comment]
Closing due to #557 merge.
